### PR TITLE
Material-style loader instead of 'loading...'

### PR DESCRIPTION
**Pull request by tholex**
similar to http://codepen.io/tholex/pen/GZvYGm but a little smaller

### CODE REVIEW

File: src/Loader/Loader.jsx [hunk 1]
```diff
@@ -7,16 +7,22 @@ class Loader extends React.Component {
...
   static propTypes = {
-    className: React.PropTypes.string
+    className: React.PropTypes.string,
+    height: React.PropTypes.string
   }
 
   render() {
+    let styleObj = {height: '60px'};
+    if (this.props.height) {
+      styleObj = {height: this.props.height };
```
Comment (learykara) [inline src/Loader/Loader.jsx]: why not `const styleObj = {height: this.props.height || '60px'};`?

Comment (learykara) [inline src/Loader/Loader.jsx]: or take care of it with `defaultProps`?

File: src/Loader/Loader.jsx [hunk 2]
```diff
@@ -7,16 +7,22 @@ class Loader extends React.Component {
...
+    }
+
     return <div
-      className={cx('tui-loader', this.props.className || "")}>
-      Loading
-      <span className="tui-loader-dot"></span>
-      <span className="tui-loader-dot"></span>
-      <span className="tui-loader-dot"></span>
+      className={cx('tui-loader', this.props.className || "")}
```
Comment (learykara) [inline src/Loader/Loader.jsx]: don't think you need the fallback `""` here

File: src/Loader/Loader.jsx [hunk 3]
```diff
@@ -7,16 +7,22 @@ class Loader extends React.Component {
...
+      style={styleObj}>
+      <svg className="tui-loader-inner" height="25" width="25">
+  <circle className="tui-loader-stroke" cx="25" cy="25" r="15" fill="none" strokeWidth="5" strokeMiterlimit="10" />
+</svg>
```
Comment (learykara) [inline src/Loader/Loader.jsx]: weird indentation from L23 - 25?

File: package.json [hunk 4]
```diff
@@ -82,7 +82,7 @@
     "react-avatar-editor": "^3.2.0",
     "react-dom": "^0.14.2",
     "react-hot-loader": "^1.2.7",
-    "reflux": "^0.2.12",
+    "reflux": "^0.3.0",
```
Comment (learykara) [inline package.json]: should this be part of this PR? doesn't seem related to reflux at all

**Comment by learykara** [comment]
:+1:

File: src/Loader/Loader.jsx (hunk 1 above)
Comment (tholex) [inline src/Loader/Loader.jsx]: went w/ defaultProps

File: src/Loader/Loader.jsx (hunk 2 above)
Comment (tholex) [inline src/Loader/Loader.jsx]: aye

File: src/Loader/Loader.jsx (hunk 3 above)
Comment (tholex) [inline src/Loader/Loader.jsx]: feexed

File: package.json (hunk 4 above)
Comment (tholex) [inline package.json]: wasn't able to `npm install` without this change
